bottomsheet/src/main/java/com/flipboard/bottomsheet/BottomSheetLayout.java

```diff
@@ -716,7 +716,6 @@ public void onAnimationEnd(Animator animation) {
                     // Remove sheet specific properties
                     viewTransformer = null;
                     onSheetDismissedListeners.clear();
```

Comment:
We should keep the behavior of `onSheetDismissedListeners` and `onSheetStateChangeListeners` the same for this, and also not clear `onSheetDismissedListeners`.